pkg/platform/kube/utils/utils.go

```diff
@@ -204,6 +204,15 @@ func GetProjectSecret(ctx context.Context, kubeClient kube.Client, projectSecret
 	return projectSecret, nil
 }
 
+func GetDefaultContainerName(annotations map[string]string) string {
+	defaultContainerAnnotation := "kubectl.kubernetes.io/default-container"
```

Comment:
This is used in [another place](https://github.com/nuclio/nuclio/blob/bec9e2fa2f9272aecdc401db53b32053db2e44ab/pkg/platform/kube/functionres/lazy.go#L1859), worth moving it to `consts.go` too